plugins/azure_sentinel/help.md

```diff
@@ -0,0 +1,613 @@
+# Description
+
+Azure Sentinel is Microsoft's' automated security service.
+
+# Key Features
+
+* Creating and updating incidents
+* Deleting incidents
+* Retriveing incident's details
+* Listing incidents for a given workspace
+* Listing bookmarks for a given incident
+* Listing alerts for a given incident
+
+# Requirements
+
+* Set of credentials with necessary permissions to monitor and modify Sentinel incidents
+
+# Supported Product Versions
+
+* 2021-04-01
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|client_id|string|None|True|The application ID that the application registration portal assigned to your app|None|c163eff0-d1a1-4618-ee2a-453534f43cee|
+|client_secret|credential_secret_key|None|True|The application secret that you generated for your app in the app registration portal|None|ef50c6bx9umaik9agvoxtoqec2fg9f0y|
+|tenant_id|string|None|True|This is active directory ID|None|5ceea899-ae8c-4ff1-fffe-353646eeeff0|
```

Comment:
"This is active directory ID" → "The Active Directory ID" [for what? from where?]